### PR TITLE
Logging tweaks

### PR DESCRIPTION
**Pull request by mortend**
Some subtle tweaks, mostly to some logging code.

### CODE REVIEW

**Comment by mortend** [comment]
This one should be good to go now...

File: src/compiler/Uno.Compiler.Core/BuildEnvironment.cs [hunk 1]
```diff
@@ -54,7 +54,7 @@ public class BuildEnvironment : LogObject, IEnvironment
             Extensions = extensions;
             Options = options;
             BundleDirectory = Path.Combine(project.CacheDirectory, "bundle");
-            CacheDirectory = Path.Combine(project.CacheDirectory, "builds", (options.BuildTarget + options.Configuration).GetHashCode().ToString("X8"));
+            CacheDirectory = Path.Combine(project.CacheDirectory, "builds", (options.BuildTarget + options.Configuration).GetHashCode().ToString("x8"));
```
Comment (kusma) [inline src/compiler/Uno.Compiler.Core/BuildEnvironment.cs]: I know the commit message says this is backwards compatible, but how come it is?

Comment (mortend) [reply src/compiler/Uno.Compiler.Core/BuildEnvironment.cs]: The cache files are regenerated automatically when not found, so renaming the directory has no other consequences than improving naming consistency.

Comment (kusma) [reply src/compiler/Uno.Compiler.Core/BuildEnvironment.cs]: Right. So you'll end up with two cache-directories until the cache gets cleared?

Comment (mortend) [reply src/compiler/Uno.Compiler.Core/BuildEnvironment.cs]: Yes.

**Pull request merged**